scripts/asciidoc2html.py

```diff
@@ -205,7 +206,7 @@ def _build_website(self) -> None:
 
         for dst, link_name in [
                 ('README.html', 'index.html'),
-                ((pathlib.Path('doc') / 'quickstart.html'),
+                ((REPO_ROOT / 'doc' / 'quickstart.html'),
```

Comment:
FWIW this was incorrect, so I reverted it in da585f2b5d6997c2f9a985eba40b69be4edfb812. See the commit message for details in case you're curious.